ws-security-stax/pom.xml

```diff
@@ -103,6 +107,10 @@
             <artifactId>bcprov-jdk15on</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.sun.xml.bind</groupId>
+            <artifactId>jaxb-impl</artifactId>
```

Comment:
Could we change all instances of this dependency to:

<groupId>org.glassfish.jaxb</groupId>
<artifactId>jaxb-runtime</artifactId>